Dockerfile

```diff
@@ -51,6 +51,7 @@ LABEL name="Nexus IQ Server image" \
 USER root
 
 # For testing
+RUN microdnf install which -y
```

Comment:
Just include this in the list of other packages being installed on line 57.

Reply:
> Just include this in the list of other packages being installed on line 57.

:+1: a507ec0a